src/Umbraco.Tests.Integration/Mapping/UmbracoMapperTests.cs

```diff
@@ -0,0 +1,360 @@
+﻿using System.Globalization;
+using System.Linq;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+using Umbraco.Core;
+using Umbraco.Core.Mapping;
+using Umbraco.Core.Models;
+using Umbraco.Core.PropertyEditors;
+using Umbraco.Core.Services;
+using Umbraco.Tests.Common.Builders;
+using Umbraco.Tests.Common.Builders.Extensions;
+using Umbraco.Tests.Integration.Testing;
+using Umbraco.Tests.Testing;
+using Umbraco.Web.Models.ContentEditing;
+
+namespace Umbraco.Tests.Integration.Mapping
+{
+    [TestFixture]
+    [UmbracoTest(Mapper = true, Database = UmbracoTestOptions.Database.NewSchemaPerTest)]
+    public class UmbracoMapperTests : UmbracoIntegrationTest
```

Comment:
There's probably good reasons why, but it doesn't feel quite right that a mapper should be an integration test - as would assume tests are checking the population of one type to another (i.e. could be a unit test all in code, and not need a database)?

Reply:
I agree, but I just migrated the existing tests. These uses the services (and db) to compare expected results..

Reply:
Understood.  For another task then.